doc/nrf/known_issues.rst

```diff
@@ -587,6 +587,15 @@ MCUboot
 Recovery with the USB does not work
   The MCUboot recovery feature using the USB interface does not work.
 
+USB
+***
+
+.. rst-class:: v1-4-99-dev1
+
+Disabling USB causes crash
```

Comment:
Is there a Jira issue for this?

Reply:
No, it is fixed upstream, but it missed v1.4.99-dev1, https://github.com/nrfconnect/sdk-nrf/pull/3460